app/grants/views.py

```diff
@@ -131,26 +143,15 @@ def grants(request):
         network=network, hidden=False, grant_type='media'
     ).count()
 
-    nav_options = [
-        {'label': 'All', 'keyword': ''},
-        {'label': 'Security', 'keyword': 'security'},
-        {'label': 'Scalability', 'keyword': 'scalability'},
-        {'label': 'UI/UX', 'keyword': 'UI'},
-        {'label': 'DeFI', 'keyword': 'defi'},
-        {'label': 'Education', 'keyword': 'education'},
-        {'label': 'Wallets', 'keyword': 'wallet'},
-        {'label': 'Community', 'keyword': 'community'},
-        {'label': 'ETH 2.0', 'keyword': 'ETH 2.0'},
-        {'label': 'ETH 1.x', 'keyword': 'ETH 1.x'},
-    ]
+    categories = [category[0] for category in basic_grant_categories()]
+
     if grant_type == 'media':
-        nav_options = [
-            {'label': 'All', 'keyword': ''},
-            {'label': 'Education', 'keyword': 'education'},
-            {'label': 'Twitter', 'keyword': 'twitter'},
-            {'label': 'Reddit', 'keyword': 'reddit'},
-            {'label': 'Blogs', 'keyword': 'blog'},
-            {'label': 'Notes', 'keyword': 'notes'},
+        categories = [
+            'education',
+            'twitter',
+            'reddit',
+            'blog',
+            'notes',
```

Comment:
shouldn't the media ones be from the backend also? I believe will be confusing for the ones creating  media grants seeing just the tech categories.

Reply:
@octavioamu Ah I see. Sorry, the media categories is a new addition to the gitcoin codebase so I hadn't previously had an opportunity to account for it. Will have to look into this further